src/utils/session.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import { INestApplication } from '@nestjs/common';
+import { getRepositoryToken } from '@nestjs/typeorm';
+import { TypeormStore } from 'connect-typeorm';
+import session from 'express-session';
+import { Repository } from 'typeorm';
+
+import { Session } from '../users/session.entity';
+
+export function createSession(app: INestApplication) {
+  app.use(
+    session({
```

Comment:
I propose to set the `name` attribute per default to something like `hedgedoc-session`.
This has particularly two reasons:
1. People get an idea what the cookie is about. It results in more trust than some cookie with an arbitrary name like `connect.sid`.
2. Running multiple nodejs apps relying on the express-session module on your local machine often results in problems because they all share the same session cookie. Setting another name per default makes development runs easier as you don't need to configure something for that.